### PR TITLE
Finished challenge

### PR DESCRIPTION
**Pull request by marcelkooi**
Please review

### CODE REVIEW

File: letsdrill.rb [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+    return "F"
+  end
+  
 
 end
 
 def shortest_string(array)
 
   #Put your code here!
+  sorted_array = array.sort_by(&:length)
```
Comment (jaybobo) [inline letsdrill.rb]: Nice. What does the syntax `&:length` mean?

File: letsdrill.rb [hunk 2]
```diff
@@ -1,12 +1,29 @@
 def get_letter_grade(integer)
  
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
```
Comment (jaybobo) [inline letsdrill.rb]: Do you need explicit return statements here?

Comment (marz0) [inline letsdrill.rb]: No I could have left it as "A" or used puts

File: letsdrill.rb (hunk 1 above)
Comment (marz0) [inline letsdrill.rb]: Correct me if I'm wrong, but the `&` converts the object to a block.  The object is the length method, denoted by the symbol `:length`, which is called on every item that `sort_by` yields

Comment (jaybobo) [inline letsdrill.rb]: You're almost there. `&` is actually `to_proc` so what you're getting is a fancy lambda - a first class function. You'll see this shorthand a lot in rails production code. Feel free to look up the difference between procs, blocks & lambdas in ruby.

You see similar things in js & python.

python
```python
map(lambda x: x+2, [1,2])
```

js
```js
[1,2].map(function(x){ return x + 2; })
```

ruby
```ruby
[1,2].map(&->(x){x*2})
[1,2].map { |x| x+2 }
```

Comment (marz0) [inline letsdrill.rb]: Nice, thanks for the feedback

File: letsdrill.rb (hunk 2 above)
Comment (degoeym) [inline letsdrill.rb]: Puts would also be unnecessary.

**Pull request closed without merging**